contrib/kind.sh

```diff
@@ -31,79 +31,82 @@ usage()
     echo "                 [-sw|--allow-system-writes] [-gm|--gateway-mode <mode>]] |"
```

Comment:
Add "-ds|--disable-snat-external-gws" to the help summary.

Reply:
ack, thx

Reply:
done

Reply:
done